bin/build

```diff
@@ -7,28 +7,7 @@ export DB=${DB:-postgresql}
 
 bin/setup
 
-echo "***********************"
```

Comment:
Similarly, could we use the [extensions format](https://github.com/solidusio-contrib/solidus_subscriptions/tree/master/bin) for the `bin` folder?